thirdparty/README.md

```diff
@@ -650,6 +650,10 @@ comments and a patch is provided in the `patches` folder.
 
 Collection of single-file libraries used in Godot components.
 
+- `bcdec.h`
+  * Upstream: https://github.com/iOrange/bcdec
+  * Version: git (026acf98ea271045cb10713daa96ba98528badb7, 2022)
```

Comment:
Just noticed this - there's been a handful of commits since (https://github.com/iOrange/bcdec/commits/main/), might be worth updating to the latest, even if there's no functional change. Just so that when I review thirdparty code I know we're up to date.